File: mycollab-dao/src/main/java/com/esofthead/mycollab/core/arguments/NoValueSearchField.java
package com.esofthead.mycollab.core.arguments;

/**
 * 
 * @author MyCollab Ltd.
 * @since 4.0
 * 
 */
public class NoValueSearchField extends SearchField {

	private String expression;
	
	public NoValueSearchField(String oper, String expression) {
		this.operation = oper;
		this.expression = expression;
	}

	public String getExpression() {
		return expression;
	}

	public void setExpression(String expression) {
		this.expression = expression;
	}
}

File: mycollab-dao/src/main/scala/com/mycollab/db/arguments/NoValueSearchField.scala
package com.mycollab.db.arguments

import scala.beans.BeanProperty

/**
  * @author MyCollab Ltd
  * @since 5.3.5
  */
class NoValueSearchField(operation: String, expression: String) extends SearchField(operation) {
  @BeanProperty val queryCount = expression
  @BeanProperty val querySelect = expression
}
